mlflow/deployments/base.py

```diff
@@ -190,3 +191,20 @@ def predict(self, deployment_name, df):
         :return: A pandas DataFrame, pandas Series, or numpy array
         """
         pass
+
+    @experimental
+    def explain(self, deployment_name, df):  # pylint: disable=unused-argument
+        """
+        Generate explanations of model predictions on the specified input pandas Dataframe
+        ``df`` for the deployed model. Explanation output formats vary by deployment target,
+        and can include details like feature importance for understanding/debugging predictions.
+        Note that we have created this function as non-abstract method to not break
+        the existing downstream.
```

Comment:
```suggestion
```

Thanks for the updated docstring! We should remove these lines IMO, I don't think we want this to show up in user-facing docs (e.g. if I'm using the deploy API I might be confused by developer-facing comments about this method being non-abstract)